torchx/lib/torchx/backend.ex

```diff
@@ -701,6 +701,42 @@ defmodule Torchx.Backend do
     end
   end
 
+  @impl true
+  def cumulative_sum(%T{type: out_type} = out, %T{} = t, axis) do
+    check_type!(out_type)
```

Comment:
@polvalente this matches `sum` and `product`, let me know if it's not what we want.